cms/core/templatetags/util_tags.py

```diff
@@ -174,3 +176,14 @@ def extend(value: list[Any], element: Any) -> None:
         # can't rely on annotations and tooling for type safety.
         raise TypeError("First argument must be a list.")
     return value.append(element)
+
+
+@register.filter(name="routablepageurl")
+def routablepageurl_no_trailing_slash(
+    context: jinja2.runtime.Context,
+    page: "Page",
+    *args: Any,
+    **kwargs: Any,
+) -> Any:
+    """Overwrite Wagtail's routablepageurl to remove trailing slash."""
+    return routablepageurl(context, page, *args, **kwargs).rstrip("/")
```

Comment:
For context: this is required because currently `routablepageurl` will always end with a slash if the `router_url` is an empty string `""` which is the case for our `latest_article` route.

https://github.com/wagtail/wagtail/blob/530182ef00672c22f7a070e1bce1b72e95545d0f/wagtail/contrib/routable_page/templatetags/wagtailroutablepage_tags.py#L26-L28

Reply:
I changed it so that the tag is conditionally registered here: [6ce7806](https://github.com/ONSdigital/dis-wagtail/pull/364/commits/6ce780640dc4dc79099b3356bd2fd20eea560540)